### PR TITLE
Unified index views

### LINKED ISSUE #91: Unify events and tasks index views
Events and View lists on index views must look unified and consistent.

### PR DESCRIPTION
**Pull request by filip373**
changed index views of projects, tasks and events to be unified,
added link to projects index in navbar

resolves #91

### CODE REVIEW

File: spec/views/events/index.html.erb_spec.rb [hunk 1]
```diff
@@ -51,14 +37,13 @@
...
-      href: "/projects/#{project.id}/events/new"
+      nil, href: "/projects/#{project.id}/events/new"
     )
   end
 
   it "renders link to project" do
     expect(rendered).to have_link(
-      "Back to project",
+      "Back",
       href: "/projects/#{project.id}"
```
Comment (crappylime) [inline spec/views/events/index.html.erb_spec.rb]: I think it can be written in one line as above

File: spec/views/project/index.html.erb_spec.rb [hunk 2]
```diff
@@ -14,36 +14,18 @@
...
-        "Edit",
-        href: "/projects/#{project.id}/edit"
-      )
-    end
-
-    it "renders link to destroy project" do
-      expect(rendered).to have_link(
-        "Destroy",
+        "Details",
         href: "/projects/#{project.id}"
```
Comment (crappylime) [inline spec/views/project/index.html.erb_spec.rb]: the same

File: spec/views/project/index.html.erb_spec.rb [hunk 3]
```diff
@@ -54,9 +36,13 @@
...
 
-    it "renders link to show project" do
+    it "renders participated projects titles" do
+      expect(rendered).to include(project.title)
+    end
+
+    it "renders link to project details" do
       expect(rendered).to have_link(
-        "Show",
+        "Details",
```
Comment (crappylime) [inline spec/views/project/index.html.erb_spec.rb]: the same

**Comment by crappylime** [comment]
:+1:
@gotar

**Comment by karolcircle** [comment]
:+1:

**Pull request merged**